keps/168-2-pending-workloads-visibility/kep.yaml

```
@@ -0,0 +1,38 @@
+title: On-demand Kueue visibility
+kep-number: 168-2
+authors:
+  - "@pbundyra"
+status: implementable
+creation-date: 2023-10-30
+reviewers:
+  - "@mwielgus"
+  - "@mimowo"
+approvers:
+  - "@alculquicondor"
+
+replaces:
+  - "/keps/168-pending-workloads-visibility"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.6"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v0.6"
+  beta: "v0.7"
+  stable: 
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: KueueVisibilityOnDemand
```

```suggestion
  - name: VisibilityOnDemand
```

No need to mention Kueue